### PR TITLE
Finish mobile section

### PR DESCRIPTION
**Pull request by Adidem-dev**
### In this section, I created:

- The middle section and
- About me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,5 +55,137 @@ <h1>
         <img src="./assets/Disabled.svg" alt="arrow icon" />
       </div>
     </section>
+
+    <section class="main-section">
+        <div class="works-section-title">
+          <h2>My Recent Works</h2>
```
Comment (ABDULALI3468) [inline index.html:61]: Please consider making the styles of this heading exactly in accordance with the Figma template to make it look more user-friendly and to meet the requirements.

## YOUR DESIGN

image


## FIGMA DESIGN

image

> Remember to copy the design of some element in FIGMA consider clicking on it and in right side panel see the styles in the inspect panel.

File: index.html [hunk 2]
```diff
@@ -55,5 +55,137 @@ <h1>
...
+          <img src="./assets/Indicator.svg" alt="" />
+        </div>
+  
+        <div class="works-grid-container">
+          <div class="card flex column">
+            <img src="./assets/card_bg.png" alt="image" />
+            <div class="card-details flex column">
+              <h2>Multi-Post-Stories</h2>
+              <h3>Gain &#43; Glory</h3>
```
Comment (ABDULALI3468) [inline index.html]: Well, these are not 2 different headings rather they need to be in a single heading like `h2`. Moreover, please consider making the styles of this heading exactly in accordance with the Figma template to make it look more user-friendly and meet the requirements 👍 


## REMEMBER:- 

This problem exists in all of the cards so consider resolving them everywhere else the next code reviewer will not approve it.

File: index.html [hunk 3]
```diff
@@ -55,5 +55,137 @@ <h1>
...
+            <h3>Skills</h3>
+            <ul class="list skills flex">
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>Web Development</li>
+              <li>API Design</li>
+            </ul>
+          </div>
+        </div>
```
Comment (ABDULALI3468) [inline index.html]: Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
<div class="about-me-container">
  <ul class="list">
    <li>
      <div class="about-me-item flex column">
        <img src="./assets/Icon Langguage.svg" class="iconic" alt="img">
        <h3>Languages</h3>
        <ul class="list skills flex">
          <li>Javascript</li>
          <li>Ruby</li>
          <li>HTML</li>
          <li>CSS</li>
        </ul>
      </div>
    </li>

    <li>
      <div class="about-me-item flex column">
        <img src="./assets/framework.svg" class="iconic" alt="img">
        <h3>Frameworks</h3>
        <ul class="list skills flex">
          <li>React.js</li>
          <li>Ruby on Rails</li>
          <li>RSpec</li>
          <li>Capybara</li>
          <li>selenium</li>
        </ul>
      </div>
    </li>

    <li>
      <div class="about-me-item flex column">
        <img src="./assets/skills.svg" class="iconic" alt="img">
        <h3>Skills</h3>
        <ul class="list skills flex">
          <li>Database Management</li>
          <li>Version Control</li>
          <li>CLI</li>
          <li>Web Development</li>
          <li>API Design</li>
        </ul>
      </div>
    </li>
  </ul>
</div>

```

Consider updating your code following this structure and rules for semantics.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Adidem-dev , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅Design is well implemented!
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbeeraTahir — APPROVED** [review]
# Status: Approved ✔️✔️✔️  

Hi @Adidem-dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

approved 

## Highlights
✔️ Linter checks are passing
✔️ Excellent implementation of the changes suggested by the previous reviewer 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**